style-guide/_colors.scss

```diff
@@ -22,13 +22,14 @@ $palette_blue_dark:          #084a67; // Safe to use with $palette_white – $pa
 $palette_green:              #77b227; // Safe to use with $palette_black.
 $palette_green_light:        #7ad03a; // Safe to use with $palette_black.
 $palette_green_medium_light: #64a60a; // Safe to use with $palette_black or $palette_white bold > 18.5px.
-$palette_green_medium:       #008a00; // Safe to use with $palette_white and $palette_black.
+$palette_green_medium:       #008a00; // Safe to use with $palette_white and $palette_black. Also used for the smiley score icon.
```

Comment:
Shouldn't we update the `color_ok` instead?

Reply:
color_good, color_ok: yes we can update them but worth noting they're used also in other places, for example in the content analysis and for the progress bar. If we want to update there too, that's fine I guess.